cargo-shuttle/src/lib.rs

```diff
@@ -112,12 +112,21 @@ impl Shuttle {
         self.run(args, provided_path_to_init).await
     }
 
-    fn find_available_port(run_args: &mut RunArgs) {
+    fn find_available_port(run_args: &mut RunArgs, services_len: usize) {
         let mut available_port = 8010;
```

Comment:
We need to start the search from `run_args.port`, we can't assume its value is 8000.